tests/SampleApps/php/exif-example/composer.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "oryx/exif-example",
+  "require": {
+    "php": ">=5.3.0",
+    "ext-exif": "*"
```

Comment:
I suppose this breaks in the build image prior to your `--ignore-platform-reqs` change?

Reply:
correct